web_external/TreeView/ActivityPane/style.styl

```diff
@@ -0,0 +1,10 @@
+.v-activity-pane
+  padding-left 25px
+
+  ul
+    list-style none
+    padding-left 20px
+
+  div.checkbox
+    margin-top 0
+    margin-bottom 1px
```

Comment:
Missing newline at end of file. (Several files have this issue; @matthewma7, can you configure your editor to not do that?)

Reply:
I can push a fix afterwards on all the files making sure that they all have end of line

Reply:
if that is okay